src/nft/components/details/detailsV2/DataPageTable.tsx

```diff
@@ -1,10 +1,15 @@
 import { Trans } from '@lingui/macro'
+import styled from 'styled-components/macro'
 
 import { ActivityTableContent } from './ActivityTableContent'
 import { ListingsTableContent } from './ListingsTableContent'
 import { OffersTableContent } from './OffersTableContent'
 import { Tab, TabbedComponent } from './TabbedComponent'
 
+export const TableContentContainer = styled.div`
+  height: 568px;
+`
```

Comment:
By defining this here, you're creating a cyclical dependency between the files using it and this file. Consider moving this to a separate file so that the build is less complex.

Reply:
Moved to the shared file